tests/protractor/page-objects/forms/delivery-report.po.js

```diff
@@ -1143,21 +1112,24 @@ module.exports = {
         return utils.saveDoc(user);
       })
       .then(done, done);
+    deleteReport(done);
```

Comment:
Looks like possible multiple calls of `done()` here.  Is it possible for `teardown` to return a `Promise` instead of using `done` at all?